src/main/java/com/terraformersmc/terrestria/surface/UluruSurfaceBuilder.java

```diff
@@ -0,0 +1,56 @@
+package com.terraformersmc.terrestria.surface;
+
+import java.util.Random;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.chunk.Chunk;
+import net.minecraft.world.gen.surfacebuilder.SurfaceBuilder;
+import net.minecraft.world.gen.surfacebuilder.TernarySurfaceConfig;
+
+public class UluruSurfaceBuilder extends SurfaceBuilder<TernarySurfaceConfig> {
```

Comment:
Can you elaborate on how exactly this is different from a normal surface builder?

Reply:
IIRC is makes the Ulurus solid sandstone.
i.e. no stone until properly below ground